Framework/PythonInterface/mantid/api/src/Exports/ITableWorkspace.cpp

```diff
@@ -657,8 +676,11 @@ void export_ITableWorkspace() {
               "Label)."))
 
       .def("getLinkedYCol", &getLinkedYCol, (arg("self"), arg("column")),
-           "Get the plot type of given column as an integer. "
-           "Accepts column name or index. ")
+           "set the data column associated with a given error column. ")
```

Comment:
Nice spot that we were missing the set method exposed. Should this say **Get** here?